libkbfs/folder_block_manager.go

```diff
@@ -889,8 +889,10 @@ func (fbm *folderBlockManager) doReclamation(timer *time.Timer) (err error) {
 	// any considerable amount of time, so it should be safe to let it
 	// run indefinitely.
 
-	// First get the current head, and see if we're staged or not.
-	head, err := fbm.helper.getMDForFBM(ctx)
+	// First get the most recent fully merged MD (might be different
+	// from the local head if journaling is enabled), and see if we're
+	// staged or not.
+	head, err := fbm.helper.getMostRecentFullyMergedMD(ctx)
 	if err != nil {
 		return err
 	} else if err := isReadableOrError(ctx, fbm.config, head.ReadOnly()); err != nil {
```

Comment:
do we still need the 3rd else if to check if the returned head is merged? Or maybe change the error to something like `returned fully-merged MD is unexpectedly unmerged`